debug/debug.go

```diff
@@ -35,7 +36,23 @@ var RecoverPrettyPanic bool
 var DeadlockDetection bool
 var WriteLogs bool
 var OnRecover func()
-var LogDirectory = filepath.Join(os.TempDir(), "gomuks-"+getUname())
+var LogDirectory = GetUserDebugDir()
+
+func GetUserDebugDir() string {
+	if runtime.GOOS == "windows" || runtime.GOOS == "darwin" {
+		return filepath.Join(os.TempDir(), "gomuks-"+getUname())
+	}
+	// See https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+	if xdgStateHome := os.Getenv("XDG_STATE_HOME"); xdgStateHome != "" {
+		return xdgStateHome
```

Comment:
Probably need to append the directory here?

```suggestion
		return filepath.Join(xdgStateHome, "gomuks")
```